app/src/org/commcare/google/services/analytics/FirebaseAnalyticsUtil.java

```diff
@@ -482,4 +482,16 @@ public static void reportNotificationType(String notificationType) {
         reportEvent(CCAnalyticsEvent.CCC_NOTIFICATION_TYPE,
                 CCAnalyticsParam.NOTIFICATION_TYPE, notificationType);
     }
+
+    public static void reportHQInvitationDeepLink(String hqDomain) {
+        reportEvent(CCAnalyticsEvent.HQ_INVITE_LINK,
+                CCAnalyticsParam.CCHQ_DOMAIN, hqDomain);
+    }
+
+    public static void reportHQInvitationResponse(String hqDomain, boolean accepted, String method) {
+        reportEvent(CCAnalyticsEvent.HQ_INVITE_LINK_RESPONSE,
+                new String[]{CCAnalyticsParam.CCHQ_DOMAIN, CCAnalyticsParam.ACCEPTED,
```

Comment:
nit: better to use `FirebaseAnalytics.Param.VALUE` to signify success as `0` or `1` . That lets us do a simple AVG() operation to get the success rate across events.